test/test_encoders.py

```diff
@@ -996,6 +1024,19 @@ def test_video_encoder_against_ffmpeg_cli(self, tmp_path, format, encode_params)
                 ff_frame, enc_frame, percentage=percentage, atol=2
             )
 
+        # Check that video metadata is the same
+        fields = ["duration", "duration_ts", "r_frame_rate", "nb_frames"]
+        ffmpeg_metadata = self._get_video_metadata(
+            ffmpeg_encoded_path,
+            fields=fields,
+        )
+        encoder_metadata = self._get_video_metadata(
+            encoder_output_path,
+            fields=fields,
+        )
+        for key in ffmpeg_metadata:
+            assert ffmpeg_metadata[key] == encoder_metadata[key]
```

Comment:
I think we should be able to just `assert ffmpeg_metadata == encoder_metadata` ?